app/model/sdl/Abstract/Model.js

```diff
@@ -666,7 +677,24 @@ SDL.SDLModel = Em.Object.extend({
         var sdl_stream = SDL.SDLController.getApplicationModel(
           appID
         ).navigationStream;
-
+        
+        console.log(' set params from VideoConfig');
+        
+        var app_model = SDL.SDLController.getApplicationModel(appID)
+        console.log("playVideo app_model", app_model)
+            
+
+        var width = SDL.SDLController.getApplicationModel(appID).VideoConfigWidth
+        var height = SDL.SDLController.getApplicationModel(appID).VideoConfigHeight 
+        
+        var marging_left = ((800 - width) / 2) 
+        var marging_top = ((400 - height) / 2) 
```

Comment:
@LuxoftAKutsan please use the same approach with width and height as I mentioned above

Reply:
72bd13d